Formula/sbt.rb

```diff
@@ -2,6 +2,8 @@ class Sbt < Formula
   desc "Build tool for Scala projects"
   homepage "https://www.scala-sbt.org/"
   url "https://piccolo.link/sbt-1.2.4.tgz"
+  mirror "https://tiny.ph/lXhU"
+  mirror "https://github.com/veera83372/homebrew-core.git"
```

Comment:
`mirror "https://github.com/veera83372/homebrew-core.git"`
Nope

Reply:
Oops In my machine it worked with first mirror. do you mind If i close this PR?

Reply:
i'll open other with proper changes